src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -281,6 +281,7 @@ The .NET Foundation licenses this file to you under the MIT license.
       <CustomLinkerArg Include="-o &quot;$(NativeBinary)&quot;" Condition="'$(TargetOS)' != 'windows'" />
       <CustomLinkerArg Include="/OUT:&quot;$(NativeBinary)&quot;" Condition="'$(TargetOS)' == 'windows'" />
       <CustomLinkerArg Include="/DEF:&quot;$(ExportsFile)&quot;" Condition="'$(TargetOS)' == 'windows' and $(ExportsFile) != ''" />
+      <CustomLinkerArg Include="/EXPORT:g_DotNetRuntimeDebugHeader,DATA" Condition="'$(OS)' == 'Windows_NT'" />
```

Comment:
Btw, would the debugger be able to find this symbol in the PDB? Do we need to export it?

If it's exported, a lot of people are going to see it so it might be "nicer" to go with Jan's original suggestion of naming it `DotNetRuntimeDebugHeader` without the `g_`. If we're expecting to keep this as our debugging future, we might as well make it not look like an accidentally exported implementation detail. It's more of a nit of course.

Reply:
If it wasn't exported I think the debugger could find it, but there are other tools outside of a debugger that we would want to support, such as the dotnet-dump tool. It's a lot easier to find as an export than having to be able to load a symbol file. I removed the "g_" from the name.